USAGE.md

```diff
@@ -0,0 +1,386 @@
+
+Please note that when you change network (different Wifi) while being connected, `/etc/resolv.con` can be updated by your system. This can cause DNS leaks, so for best security, use `protonvpn reconnect` after changing network.
+
+**Enabling DNS Leak Protection**
+
+To enable DNS Leak Protection use the `protonvpn configure` command, then press `4` to choose DNS Management. Then press `1` to choose that you want to enable DNS Leak Protection.
+
+From now on your DNS queries are secured.
+
+#### Custom DNS
+
+If you always want to use a custom DNS Server when you connect to ProtonVPN, ProtonVPN-CLI allows you to do this as well. You can add up to 3 custom DNS Servers.
+
+**Enabling Custom DNS**
+
+To enable DNS Leak Protection use the `protonvpn configure` command, then press `4` to choose DNS Management. Then press `2` to choose that you want to configure custom DNS Servers. Now enter the IPs of up to 3 DNS Servers you want to use and confirm with Enter.
+
+#### Disabling DNS Management
+
+If you don't want ProtonVPN-CLI to do any changes to your DNS, you can do this as well. This will cause ProtonVPN-CLI to not touch `/etc/resolv.conf` and your device will always use the DNS servers configured by you or through your network.
+
+**Disabling any DNS management**
+
+To enable DNS Leak Protection use the `protonvpn configure` command, then press `4` to choose DNS Management. Then press `3` to disable any DNS management.
+
+### IPv6 Leak Protection
+
+ProtonVPN-CLI features an IPv6 Leak Protection feature. It makes sure that your IPv6 address doesn't get leaked when connecting to a ProtonVPN server.
+
+This feature is enabled by default and for security reasons can't be disabled.
+
+It works by detecting the IPv6 address, backing it up and removing it from the default interface. When disconnecting it adds the IPv6 address back to the default interface and deleting the backup.
+
+### Kill Switch
+
+ProtonVPN-CLI has a built-in Kill Switch that protects your data in a case where your VPN connection would go down unexptectedly.
+
+It works by replacing your existing iptables rules with custom rules that only allow data to go over the OpenVPN interface when the VPN connection is established. When disconnecting ProtonVPN-CLI will revert iptables to it's previous state.
+
+**Enabling the Kill Switch**
+
+To enable the Kill Switch, open the configuration menu with `protonvpn configure`, then select `5` for the Kill Switch and confirm the activation with `y`. On the next connection the Kill Switch will be enabled.
+
+*Note: The Kill Switch only activates on unexptected connection drops. It will not persist through reboots and not activate when calling `protonvpn disconnect`. To simulate the Kill Switch, kill the OpenVPN process while connected with `sudo pkill openvpn`.*
+
+### Split Tunneling
+
+ProtonVPN-CLI features IP-based split tunneling. This means that you can exclude specific IPs or networks from being routed over the VPN tunnel.
```

Comment:
This might be more clear if instead of saying networks, you just said IP ranges. "This means that you can exclude specific IPs or IP ranges from being routed over the VPN tunnel."